dashboard/app/controllers/pd/workshop_enrollment_controller.rb

```diff
@@ -90,43 +90,6 @@ def new
     end
   end
 
-  # POST /pd/workshops/1/enroll
-  def create
-    @workshop = ::Pd::Workshop.find_by_id params[:workshop_id]
-    if @workshop.nil?
-      render_404
-      return
-    end
-
-    enrollment_email = enrollment_params[:email]
-    user = User.find_by_email_or_hashed_email enrollment_email
-
-    # See if a previous enrollment exists for this email
-    previous_enrollment = @workshop.enrollments.find_by(email: enrollment_email)
-    if previous_enrollment
-      @cancel_url = url_for action: :cancel, code: previous_enrollment.code
-      render :duplicate
-    elsif workshop_owned_by? user
-      render :own
-    elsif workshop_closed?
-      render :closed
-    elsif workshop_full?
-      render :full
-    else
-      @enrollment = ::Pd::Enrollment.new workshop: @workshop
-
-      @enrollment.school_info_attributes = school_info_params
-
-      if @enrollment.update enrollment_params
-        Pd::WorkshopMailer.teacher_enrollment_receipt(@enrollment).deliver_now
-        Pd::WorkshopMailer.organizer_enrollment_receipt(@enrollment).deliver_now
-        redirect_to action: :thanks, code: @enrollment.code, controller: 'pd/workshop_enrollment'
```

Comment:
If we remove this line, then I'm good to remove the `thanks` method and partial, but again, I think I've seen this recently when testing